api.md

```diff
@@ -710,6 +725,70 @@ SELECT set_number_partitions('conditions', 2, 'device_id');
 
 ---
 
+## show_chunks() [](show_chunks)
+Get list of chunks associated with hypertables.
+
+#### Optional Arguments [](show_chunks-optional-arguments)
+
+Function accepts the following timestamp arguments. When both timestamp arguments
+are used, the function returns the intersection of those two timestamp ranges.
+
+|Name|Description|
+|---|---|
+| `older_than` | Timestamp of cut-off point where any full chunks older than this timestamp should be shown. |
+| `newer_than` | Timestamp of cut-off point where any full chunks newer than this timestamp should be shown. |
+
+The `older_than` and `newer_than` parameters can be specified in two ways:
+
+- **interval type**: The cut-off point is computed as `now() -
+    older_than`.  An error will be returned if an INTERVAL is supplied
```

Comment:
can we verify that if newer_than as interval is also now()-newer_than and make that clear in the docs.

Reply:
Yes, looks like SELECT show_chunks(newer_than => now() - interval '100 months'); returns the same thing. I'll also make some copy edits to make this clearer

Reply:
Done!